src/StratisMasternodeDashboard/Controllers/HomeController.cs

```diff
@@ -92,6 +93,22 @@ public IActionResult Index()
             else
                 this.ViewBag.Status = "OK";
 
+            SideChainNodeStatsModel sideChainNodeStats = new SideChainNodeStatsModel();
+            if (string.IsNullOrEmpty(this.distributedCache.GetString("SideChainNodeStats")))
```

Comment:
Instead of getting the value from the cache twice, add a variable above this line that holds the cached vaue and then check if its null or not. You can then just use the value in the `else`.